template/pkg.tpl

```diff
@@ -0,0 +1,49 @@
+{{ define "packages" }}
```

Comment:
Since we are downloading the .tar.gz package, and it already has the template folder, I have completely removed the folder from our project, and I am using in-build flag to locate the template folder and utilize it.